react/features/toolbox/components/ToolbarButton.web.js

```diff
@@ -1,19 +1,30 @@
 /* @flow */
 
+import AKInlineDialog from '@atlaskit/inline-dialog';
```

Comment:
I don't think that applies to default imports. I just did some reading on them and you can give an alias to the default. For example, this line could be changed to "import Popup" and still work.